doc/plotting.rst

```diff
@@ -66,14 +66,18 @@ For these examples we'll use the North American air temperature dataset.
     # Convert to celsius
     air = airtemps.air - 273.15
 
+    # copy attributes to get nice figure labels and change Kelvin to Celsius
+    air.attrs = airtemps.air.attrs
+    air.attrs['units'] = 'deg C'
+
 
 One Dimension
 -------------
 
 Simple Example
 ~~~~~~~~~~~~~~
 
-xarray uses the coordinate name to label the x axis.
+xarray uses the coordinate name along with any associated `CF-compliant metadata <http://cfconventions.org/Data/cf-conventions/cf-conventions-1.7/build/ch03s03.html>`_ (the `attrs` 'long_name', 'standard_name' and 'units') to label the axes.
```

Comment:
Can you elaborate a little bit more on how these are used and what the expectations are? It would be good to entirely self-contained, so users don't need to reference CF conventions (which are long and mostly irrelevant to this)

Reply:
Yes, I've added a new commit. Let me know if you want more changes.